tests/playwright-test/ui-mode-trace.spec.ts

```diff
@@ -197,3 +197,32 @@ test('should show screenshot', async ({ runUITest, server }) => {
   await expect(page.getByText('Screenshots', { exact: true })).toBeVisible();
   await expect(page.locator('.attachment-item img')).toHaveCount(1);
 });
+
+test('should not fail on internal page logs', async ({ runUITest, server }) => {
```

Comment:
Ideally, this would be a library test.

Reply:
It is tricky to make it a library test because the code path only triggers via @playwright/test.